tsconfig.json

```
@@ -23,7 +23,7 @@
   "angularCompilerOptions": {
     "fullTemplateTypeCheck": true,
     "strictInjectionParameters": true,
-    "enableIvy":false
+    "enableIvy":true
```

Hi @eneufeld ,

I tried to run this pull request branch, but the error below is shown:

`    
 ERROR in Symbol BooleanControlRenderer declared in /home/.../git/jsonforms-angular-material-playground/node_modules/@jsonforms/angular-material/lib/controls/boolean.renderer.js is not exported from /home/.../git/jsonforms-angular-material-playground/node_modules/@jsonforms/angular-material/lib/controls/boolean (import into /home/.../git/jsonforms-angular-material-playground/node_modules/@jsonforms/angular-material/lib/module.js)
`

`
ERROR in node_modules/@jsonforms/angular-material/lib/module.d.ts:1:22 - error NG6002: Appears in the NgModule.imports of AppModule, but could not be resolved to an NgModule class
   
    1 export declare class JsonFormsAngularMaterialModule {
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
`

To fix it I changed Ivy to disabled.

```suggestion
    "enableIvy":false
```

---

which version of jsonforms did you use?
Could you test this with 2.4.0-alpha.1?

---

I used 2.4.0-alpha.0:

https://github.com/eclipsesource/jsonforms-angular-material-playground/pull/34/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R28

I changed to  2.4.0-alpha.1 and it works:
https://github.com/piantino/jsonforms-angular-material-playground/commit/5e44f11fd142f6b656c092bb07ac36e1ba8318e3